win32ss/user/ntuser/input.c

```diff
@@ -331,20 +331,40 @@ RawInputThreadMain(VOID)
 }
 
 /*
- * CreateSystemThreads
+ * UserSystemThreadProc
  *
  * Called form dedicated thread in CSRSS. RIT is started in context of this
  * thread because it needs valid Win32 process with TEB initialized.
  */
 DWORD NTAPI
-CreateSystemThreads(UINT Type)
+UserSystemThreadProc(BOOL bRemoteProcess)
 {
+    DWORD Type;
+
+    ASSERT(gdwPendingSystemThreads);
```

Comment:
Since this is called from user mode, you'll want to gracefully handle this and return failure if called too often.